src/coreos-assembler

```diff
@@ -41,7 +41,7 @@ fi
 
 cmd=${1:-}
 # commands we'd expect to use in the local dev path
-build_commands="init fetch build run prune clean list"
+build_commands="init fetch build run prune prune-pkgcache clean list"
```

Comment:
Perhaps rather than a new command, this is just `prune --pkgcache`?

Reply:
Thank you for review. This idea is great. I just have one small doubt. `prune --pkgcache` removes previous builds, while this PR is to delete refs packages. Do they sound like related things if we use `prune --pkgcache` ?